sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -215,7 +185,7 @@ public void testRead() throws Exception {
     PCollection<KV<String, Integer>> output = pipeline.apply(
         JdbcIO.<KV<String, Integer>>read()
             .withDataSourceConfiguration(JdbcIO.DataSourceConfiguration.create(dataSource))
-            .withQuery("select name,id from BEAM")
+            .withQuery("select name,id from " + JdbcTestDataSet.READ_TABLE_NAME)
```

Comment:
remain consistent about `?` vs string concat (here vs L226)? I suspect `?` is better.